packages/sdk/src/types/AsyncActions.ts

```diff
@@ -0,0 +1,24 @@
+export interface AsyncActionsResponse {
+	id: string
+	type: string
+	status: 'ENQUEUED' | 'IN_PROGRESS' | 'DONE' | 'FAILED'
+	result: {
```

Comment:
this will be a dynamic field - it's content will depend on AA type,
I think we should make it a generic json, and probably leave it like that